pkgs/servers/mail/dovecot/default.nix

```diff
@@ -61,6 +76,9 @@ stdenv.mkDerivation rec {
       libcap
       inotify-tools
     ]
+    ++ lib.optional (stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isDarwin) libtirpc
+    ++ lib.optional (stdenv.hostPlatform.isLinux && withApparmor) libapparmor
```

Comment:
```suggestion
    ++ lib.optional withApparmor libapparmor
```
we normally do this to explicitly honor the users override

Reply:
Thanks, fixed.